src/main/java/edu/oregonstate/cs361/battleship/BattleshipModel.java

```diff
@@ -31,7 +31,7 @@
 	private ArrayList<Coord> playerHits = new ArrayList<Coord>();
 	private ArrayList<Coord> playerMisses = new ArrayList<Coord>();
 	private boolean scanResult = false;
-	private boolean difficulty = false; //False means easy, true for hard mode
+	private boolean difficulty = true; //False means easy, true for hard mode

```

Comment:
Will review this implementation and see if some changes need to be made since i originally coded difficulty as a string = ("easy"/"hard") .